iexec-sms-library/src/main/java/com/iexec/sms/SmsClient.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.sms;
+
+import feign.Headers;
+import feign.Param;
+import feign.RequestLine;
+
+/**
+ * Interface allowing to instantiate a Feign client targeting SMS REST endpoints.
+ * <p>
+ * To create the client, call:
+ * <pre>FeignBuilder.createBuilder(feignLogLevel)
+ *         .target(SmsClient.class, smsUrl)</pre>
+ * @see com.iexec.common.utils.FeignBuilder
+ */
+public interface SmsClient {
+
+    @RequestLine("POST /apps/{appAddress}/secrets/{secretIndex}")
```

Comment:
Minor: we are used to `@PostMapping(...)`. Is there any benefit in using `@RequestLine("POST ...")`? If there's no real benefit, we should probably stay consistent with `@PostMapping`.

Reply:
Right, thanks!